dev-aws/finance/topics.tf

```diff
@@ -0,0 +1,16 @@
+resource "kafka_topic" "dd_run_events" {
+  name               = "dd-run.events"
+  replication_factor = 3
+  partitions         = 10
+}
+
+resource "kafka_topic" "inovice_payment_events" {
+  name               = "invoice.payment.events"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    "retention.bytes" : "-1"
+    # keep for 30 days as it is indexed
+    "retention.ms" : "2592000000"
```

Comment:
Do we want to rely on default values for the rest of the config?

Reply:
https://github.com/utilitywarehouse/kubernetes-manifests/blob/master/dev-aws/finance/kafka/kafka-topic-applier/topics.yaml#L319

Reply:
I copied the config from the old cluster. Which bit do you think is missing?

Reply:
Fair enough, I only looked at `cleanup_policy: delete` missing